### DISCUSSION TITLE
Save Island missions into global db

### DISCUSSION THREAD

**Discussion by nivcoo** [BG-Software-LLC/SuperiorSkyblock2#598] [Denied Feature Requests] [answered]
Hello, idk why but the mission module saved all progress into yml file, can you migrate saves into db ? (I've lot of issue with yml, for example when the server crash or restart sometimes the datas isn't save). Thx :p

**Comment by OmerBenGera** [comment]
This is a bit of a problem because missions are hard-coded designed to be saved into a YAML file.
I think it's possible tho thanks to the new database system
I will see what can be done regarding this.

**Reply by nivcoo** [reply]
Thx you

**Comment by OmerBenGera** [comment, marked as answer]
For now, I will keep it in yaml as missions can have custom data, etc.
I am opened for PRs :)
